File: Ejercicio6-Expresiones.cpp
//6. Escriba un programa que lea las tres notas de un alumno y calcule la nota final media de dicho alumno.

#include<iostream>
using namespace std;

int main (){
    float nota1, nota2, nota3, media = 0;

    cout<<"Digite la calificacion 1: "; cin>>nota1;
    cout<<"Digite la calificacion 2: "; cin>>nota2;
    cout<<"Digite la calificacion 3: "; cin>>nota3;

    media=(nota1+nota2+nota3)/3;

    cout.precision(2);
    cout<<"\nLa media de las calificaciones es:  "<<media<<endl;
    return 0;
}

File: Comparativa Python/Ejercicio6-Expresiones.py
"""6. Escriba un programa que lea las tres notas de un alumno y calcule la
nota final media de dicho alumno."""

nota1 = float(input('Digite nota 1: '))
nota2 = float(input('Digite nota 2: '))
nota3 = float(input('Digite nota 3: '))

calificacion = (nota1+nota2+nota3)/3

print('La nota final es: ', str(round(calificacion, 2)))
